src/Powercord/plugins/pc-moduleManager/components/parts/BaseProduct.jsx

```diff
@@ -2,12 +2,23 @@ const { React, getModule, constants: { Routes }, i18n: { Messages } } = require(
 const { Divider, Button } = require('powercord/components');
 
 const { shell: { openExternal, openPath } } = require('electron');
-
+const path = require('path');
 const Details = require('./Details');
 const Permissions = require('./Permissions');
 const TIMEOUT = 10e3;
 
 class BaseProduct extends React.PureComponent {
+  constructor (props) {
+    super(props);
+    this.state = {
+      gitInfo: null
+    };
+  }
+
+  async componentDidMount () {
+    await this.getGitInfo(this.props.Path).then(i => this.setState({ gitInfo: i }));
```

Comment:
same thing with the casing here, although I don't blame you because it was like this way before :/

could you change it anyways? (and references to this prop)